consensus/istanbul/core/commit.go

```diff
@@ -79,10 +79,6 @@ func (c *core) handleCommit(msg *message, src istanbul.Validator) error {
 	}
 
 	//logger.Error("receive handle commit","num", commit.View.Sequence)
-	if !c.valSet.CheckInSubList(msg.Hash, commit.View, src.Address()) {
```

Comment:
So what you intended is to remove false alarm for future messages caused by `CheckInSubList`? Did I understand correctly?

Reply:
Maybe I didn't explain well. `CheckInSubList` in here is removing some positive alarm for future messages because `CheckInSubList` may invalidate future messages. 
By moving `CheckInSubList` bottom, positive future messages will be handled correctly in `checkMessage`.